src/referee.cpp

```diff
@@ -3099,6 +3110,14 @@ HFORef::analyse()
             {
                 M_take_time = 0;
             }
+
+            // Broadcast who has possession. Goalie catches are broadcast separately by the server. Rebroadcasting a message here would overwrite the other one. 
```

Comment:
Do we need to broadcast every step or can we just broadcast when a new player gets possession of the ball?